### ISSUE TITLE
MRI-ESM2-0-tasmax.yaml fails cleaning step for ssp126 and ssp585 due to "missing time steps"

### ISSUE THREAD

**Issue by brews** [ClimateImpactLab/downscaleCMIP6#356]
MRI-ESM2-0-tasmax.yaml failed an end-to-end run due to ssp126 and ssp585 cleaning step validation (https://argo.cildc6.org/workflows/default/e2e-mri-esm2-0-tasmax-9xmjn).

Blocks progress on run #206.

Log for cleaning validation for ssp126:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/MRI/MRI-ESM2-0/ssp126/r1i1p1f1/day/tasmax/gn/v20191108.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 645, in _test_timesteps
    assert (
AssertionError: projection cmip6 file is missing timesteps, only has 108405
```

log for cleaning validation step for ssp585:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/MRI/MRI-ESM2-0/ssp585/r1i1p1f1/day/tasmax/gn/v20191108.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 645, in _test_timesteps
    assert (
AssertionError: projection cmip6 file is missing timesteps, only has 108405
```

**Comment by emileten** [comment]
@brews this is because it was not yet using https://github.com/ClimateImpactLab/dodola/pull/140 right ?

**Comment by brews** [comment]
@emileten Yeah, it's *likely* going to be fixed by https://github.com/ClimateImpactLab/dodola/pull/140, but we won't know until we try it.

I think this was also run last week. I just didn't have the time to parse the errors until now.

**Comment by brews** [comment]
Closed with PR #365
